### PR TITLE
Updating NDX to 1.0.0-rc1-update1

### PR DESCRIPTION
**Pull request by alpaix**
- Merged @emgarten xunit changes from PR #118 
- Changed build.ps1 script to always install new ndx version
- Upgraded xunit runner dependency to match ndx version (from here https://xunit.github.io/docs/getting-started-dnx.html)
- Resolved deadlock in command line unit-tests.

### CODE REVIEW

File: src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs [hunk 1]
```diff
@@ -120,7 +120,12 @@ public void Execute()
...
                 UserAgent.UserAgentString = UserAgent.CreateUserAgentString(CommandLineConstants.UserAgent);
 
-                ExecuteCommandAsync().Wait();
+                // Starts the asynchronous task on a thread-pool thread.
+                // Returns a proxy to the original task.
+                var task = Task.Run(() => ExecuteCommandAsync());
+
+                // Will block until the task is completed...
+                task.Wait();
             }
```
Comment (alpaix) [inline src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs:124]: This is the magic that sends async task to a threadpool while the main thread is blocked on waiting.

**Comment by emgarten** [comment]
Looks good :shipit:

**Comment by davidfowl** [comment]
Is this getting merged soon?

**Comment by alpaix** [comment]
Merged into dev.
Thanks for review!

**Pull request merged**